tests/models/soil/test_carbon.py

```diff
@@ -81,17 +33,23 @@ def test_calculate_soil_carbon_updates(dummy_carbon_data, top_soil_layer_index):
         soil_c_pool_maom=dummy_carbon_data["soil_c_pool_maom"].to_numpy(),
         soil_c_pool_microbe=dummy_carbon_data["soil_c_pool_microbe"].to_numpy(),
         soil_c_pool_pom=dummy_carbon_data["soil_c_pool_pom"].to_numpy(),
+        soil_enzyme_pom=dummy_carbon_data["soil_enzyme_pom"].to_numpy(),
+        soil_enzyme_maom=dummy_carbon_data["soil_enzyme_maom"].to_numpy(),
         pH=dummy_carbon_data["pH"],
         bulk_density=dummy_carbon_data["bulk_density"],
-        soil_moisture=np.array([0.5, 0.7, 0.6, 0.2]),
+        soil_moisture=dummy_carbon_data["soil_moisture"][
+            top_soil_layer_index
+        ].to_numpy(),
         soil_water_potential=dummy_carbon_data["matric_potential"][
             top_soil_layer_index
         ].to_numpy(),
+        vertical_flow_rate=dummy_carbon_data["vertical_flow"] / 30.0,
         soil_temp=dummy_carbon_data["soil_temperature"][top_soil_layer_index],
```

Comment:
Add a TODO here to remove the `/ 30.0` once the vertical flow is changed to mean flow as discussed here #347